app/controllers/api/database.php

```diff
@@ -734,13 +698,7 @@ function createAttribute(string $collectionId, Document $attribute, Response $re
     ->inject('audits')
     ->inject('usage')
     ->inject('events')
-    ->action(function ($collectionId, $key, $size, $required, $default, $array, $response, $dbForProject, $database, $audits, $usage, $events) {
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Database $dbForProject*/
-        /** @var Appwrite\Event\Database $database */
-        /** @var Appwrite\Event\Audit $audits */
-        /** @var Appwrite\Stats\Stats $usage */
-        /** @var Appwrite\Event\Event $events */
+    ->action(function (string $collectionId,string $key, int $size, bool $required, ?string $default, bool $array, Response $response, Database $dbForProject, EventDatabase $database, EventAudit $audits, Stats $usage, Event $events) {
```

Comment:
Shouldn't $size and $required be nullable here ?

Reply:
updated